tests/phpunit/Metrics/CalculatorTest.php

```diff
@@ -139,16 +140,16 @@ public function metricsProvider(): iterable
             2,
             2,
             0,
-            84.61538461538461,
+            84.62,
```

Comment:
we can leave on of the data set with previous precision (84.61538461538461) to test it still works if user wants it